sit-common/src/main/java/com/sit/common/api/interceptor/AuthInterceptor.kt

```diff
@@ -0,0 +1,21 @@
+package com.sit.common.api.interceptor
+
+import com.sit.common.preference.CommonPreferenceManager
+import com.sit.common.utils.Constant
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+
+class AuthInterceptor(
+    private val preferenceManager: CommonPreferenceManager
+) : Interceptor {
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val newRequest: Request.Builder = chain.request().newBuilder()
+        if (preferenceManager.authToken.isNotEmpty()) {
+            newRequest.addHeader(Constant.AUTHORIZATION, preferenceManager.authToken)
```

Comment:
Please use strings from the strings.xml file so we can adjust the messages according to the app's needs.